src/ServiceManagement/Compute/Commands.ServiceManagement.Test/UnitTests/Cmdlets/IaaS/Extensions/Diagnostics/DiagnosticsHelperTest.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using Microsoft.WindowsAzure.Commands.ServiceManagement.Common;
+using Xunit;
+
+namespace Microsoft.WindowsAzure.Commands.ServiceManagement.Test.UnitTests.Cmdlets.IaaS.Extensions.Diagnostics
+{
+    public class DiagnosticsHelperTest
+    {
+        [Fact]
```

Comment:
I only see BVT defined in this project which looks closer.
Here is what marked for DSC extension, [Trait(Category.Functional, Category.BVT)]. Should I follow this attribute or add "CheckIn" in the category?